src/app/channels/channels.component.css

```
@@ -1,11 +1,28 @@
-.operation-badge {
-    background-color: #CDDC39;
+
+.badge {
     border-radius: 4px;
     padding: 8px;
     font-size: small;
     font-weight: bold;
 }
 
+.subscribe-badge {
+    background-color: #FFD580;
+}
+
+.publish-badge {
+    background-color: #9BD279;
+}
+
+.protocol-badge {
+    background-color: #347AEB;
+    color: #fff
+}
+
+.fa {
```

Such a generic class will apply to font-awesome icons on the page.
I suggest making it a bit more selective

```suggestion
.badge .fa {
```